client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-footer.jsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { actionType } from './constants';
+
+class DomainConnectAuthorizeFooter extends Component {
+	static propTypes = {
+		isPlaceholder: PropTypes.bool,
+		onClose: PropTypes.func,
+		onConfirm: PropTypes.func,
+		showAction: PropTypes.oneOf( actionType )
```

Comment:
Expects array, you're returning an object. You can probably limit to string or do `keys( actionType )`.